polaris.shopify.com/content/components/box.md

```diff
@@ -1,27 +1,39 @@
 ---
 title: Box
-description: Box is the most primitive layout component. Box has a set of padding options. Use it to render an individual item.
+description: Box is the most primitive layout component. It’s a way to access Polaris design tokens.
 category: Structure
 keywords:
   - layout
 status:
   value: Alpha
   message: This component is in development. There could be breaking changes made to it in a non-major release of Polaris. Please use with caution.
 examples:
-  - fileName: box-with-padding.tsx
-    title: Padding
+  - fileName: box-default.tsx
+    title: Default
+    description: >-
+      By default, box has no padding, borders, background, or shadows, and border radius is set to zero.
+  - fileName: box-with-color.tsx
+    title: Color
+    description: >-
+      Background color of box and text color inside a box can be adjusted using the [Color tokens](http://localhost:3000/tokens/colors).
+  - fileName: box-with-border.tsx
+    title: Border
     description: >-
-      Spacing inside Box can be set with the padding properties. Padding options are provided using our spacing tokens. There are 5 different padding properties: padding, paddingLeft, paddingRight, paddingTop, and paddingBottom.
+      Border width can be adjusted using the [Shape tokens](https://polaris.shopify.com/tokens/shape), and a subset of tokens allows different types of border.
   - fileName: box-with-border-radius.tsx
     title: Border radius
     description: >-
-      Border radius can be set with the border radius properties. Border radius options are provided using our shape tokens. There are 5 different padding properties: borderRadius, borderRadiusLeft, borderRadiusRight, borderRadiusTop, and borderRadiusBottom.
-  - fileName: box-with-background.tsx
-    title: Background
+      Border radius can be adjusted using the [Shape tokens](https://polaris.shopify.com/tokens/shape).
+  - fileName: box-with-padding.tsx
+    title: Padding
     description: >-
-      Background color for Box can be set with the background prop. Background color options are provided using our color tokens that relate to background, surface, backdrop, and overlay.
-  - fileName: box-with-border.tsx
-    title: Border
+      Padding can be added to either all sides, left, right, top, and bottom, using the [Spacing tokens](http://localhost:3000/tokens/spacing).
+  - fileName: box-with-shadow.tsx
+    title: Shadow
     description: >-
-      Border color for Box can be set with the border prop. Border color options are provided using our shape tokens.
+      Shadow can be applied using the Shadow token. The base is --p-shadow-base.
```

Comment:
I guess we should link the Shadow token, sorry should have noticed it sooner. Thank you! 

https://polaris.shopify.com/tokens/shape

Reply:
Added the Depth hyper link: https://polaris.shopify.com/tokens/depth